### ISSUE TITLE
Slideshow: Add labels to navigation buttons for accessibility

### ISSUE THREAD

**Issue by juliasydnor** [Automattic/jetpack#13603]
The slideshow block has labels on the navigation buttons for accessibility, but slideshows created with the Calypso Classic editor and/or `[gallery type=”slideshow”]` shortcode do not.

From user, both slideshows should have the same type of "meaningful text" on the navigation buttons.

Classic Editor / Shortcode Slideshow
![Screen Shot on 2019-09-30 at 15:03:11](https://user-images.githubusercontent.com/29981758/65908087-a76ac200-e393-11e9-977f-9b58464befbd.png)

Slideshow Block
![Screen Shot on 2019-09-30 at 14:59:42](https://user-images.githubusercontent.com/29981758/65908105-b3568400-e393-11e9-93d3-b7f880a47d5b.png)

A related accessibility change to the slideshow buttons was done here: https://github.com/Automattic/jetpack/issues/1001

**Comment by CarlyGerard** [comment]
Hi @jeherve, I was the person that initially made this request, and was hoping this could be a quick patch. Our WordPress hosting provider is not yet moving to Gutenberg (where this issue is fixed), and this is the only issue where if resolved, would make this player a reasonable alternative to other inaccessible slideshows we're trying to get rid of to improve our overall web accessibility.

I saw you assigned this a low priority, but would it be possible to make it a higher priority for those that can't support Gutenberg just yet? If it's not possible it's not, but wanted to clarify the reason for this ticket and see if this could be a higher priority. Thank you.

**Comment by jeherve** [comment]
We're unfortunately not really focussed on the old slideshow shortcode right now; we're not planning on iterating on this shortcode in the near future. 

That said, that doesn't mean this can't be fixed. I've added a few more labels to this issue to give it more visibility. If anyone is willing to contribute a fix for the Slideshow shortcode (currently located here: https://github.com/Automattic/jetpack/blob/89a9af96b669e2e5a2ed47d3f3e07c804d6e0dd0/modules/shortcodes/slideshow.php ), we'll be happy to review the patch and include it in a future version of Jetpack!

**Comment by codestor4** [comment]
I'm giving this a try (submitted a PR above).

**Comment by scottsweb** [comment]
Looks like the [PR above](https://github.com/Automattic/jetpack/pull/13739) was merged and this issue is now fixed. Closing.
